Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -257,3 +257,14 @@ The full ``CarAdmin.php`` example looks like this:
             // ...
         }
     }
+
+.. note::
+
+If you want to render a custom controller action in a template by using the
+render function in twig you need to add ``_sonata_admin`` as an attribute. For
+example; ``{{ render(controller('AppBundle:XxxxCRUD:comment', {
+'_sonata_admin': 'sonata.admin.xxxx' })) }}``. This has to be done because the
+moment the rendering should happen the routing, which usually sets the value of
+this parameter, is not involved at all, and then you will get an error 'There
+is no _sonata_admin defined for the controller
+AppBundle\Controller\XxxxCRUDController and the current route ''.'
```

Comment:
The ''.' an empty route, otherwise the error would have displayed the route name inside the first two quotes. The last quote after the full stop is the quote quoting the entire error. You'll see the first quote before the 'there', but I'll use double quotes instead.